pegasus/sites.v3/code.org/public/student/elementary.haml

```diff
@@ -4,71 +4,148 @@ theme: responsive
 ---
 
 %link{type: "text/css", href: "/css/learn-carousel.css", rel: "stylesheet"}
-= view 'mobilecheck.js'
-
-.solid-block-header
-  Grades K-5
-
-%h1 Courses by Code.org
-
-%h2 Computer Science Fundamentals Express Courses
-
-A great option for students getting started on their own.  You'll learn the
-fundamentals of computer science with drag & drop blocks.  Create your own
-drawings and games.
-
-%div{style: "clear: both"}
-%br
-
-.col-50{style: "padding-right: 10px;"}
-  %a{href: CDO.studio_url('/s/pre-express')}
-    %img{src: "/images/curriculum/course-tile-pre-reader-express.png", style: "max-width: 100%;"}
-
-.col-50{style: "padding-right: 10px;"}
-  %a{href: CDO.studio_url('/s/express')}
-    %img{src: "/images/curriculum/course-tile-express.png", style: "max-width: 100%;"}
-
-%div{style: "clear: both"}
-
-%h2 Computer Science Fundamentals for Elementary Schools
-
-For pre-readers in elementary school classrooms
-
-%div{style: "clear: both"}
-
-.col-25
-  %a{href: CDO.studio_url('/s/coursea')}
-    %img{src: "/images/curriculum/new-course-tile-courseA.png"}
-
-.col-25
-  %a{href: CDO.studio_url('/s/courseb')}
-    %img{src: "/images/curriculum/new-course-tile-courseB.png"}
-
-%div{style: "clear: both"}
-%br
-%br
-
-For older students in elementary school classrooms
-
-%div{style: "clear: both"}
-
-.col-25
-  %a{href: CDO.studio_url('/s/coursec')}
-    %img{src: "/images/curriculum/new-course-tile-courseC.png"}
-
-.col-25
-  %a{href: CDO.studio_url('/s/coursed')}
-    %img{src: "/images/curriculum/new-course-tile-courseD.png"}
-
-.col-25
-  %a{href: CDO.studio_url('/s/coursee')}
-    %img{src: "/images/curriculum/new-course-tile-courseE.png"}
-
-.col-25
-  %a{href: CDO.studio_url('/s/coursef')}
-    %img{src: "/images/curriculum/new-course-tile-courseF.png"}
-
-%div{style: "clear: both"}
-
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/student/school-styles.css", rel: "stylesheet"}
 
+- js_locale = request.locale.to_s.downcase.tr('-', '_')
+%script{src: webpack_asset_path("js/#{js_locale}/common_locale.js")}
+%script{src: webpack_asset_path('js/code.org/public/student/middle-high.js')}
+%link{href: "/shared/css/course-blocks.css", rel: "stylesheet", type: "text/css"}
+= view 'mobilecheck.js'
 
+%h1
+  =hoc_s(:elemeentary_page_main_title)
+%p{style: "font-size: 20px;"}
+  =hoc_s(:elementary_page_main_description)
+
+%section 
+  %h2
+    =hoc_s(:middle_high_page_hero_title)
+  %p
+    =hoc_s(:middle_high_page_hero_description)
+  .action-block.action-block--two-col.video-101__text
+    .video-101__image
+      %img{src: "/images/houreofcode-students-banner.png", style: "width: 100%" }
+    .text-wrapper.col-50
+      %p.overline
+        all ages
```

Comment:
This needs to be a string and updated in the other places where `All Ages` is used.